NuGet.config

```diff
@@ -12,6 +12,7 @@
     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
     <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
     <add key="vs-buildservices" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <add key="vssdk-unifiedPIAs" value="https://pkgs.dev.azure.com/devdiv/DevDiv/_packaging/vssdk-unifiedPIAs/nuget/v3/index.json" />
```

Comment:
Using this feed means PR/CI build would fail, since our pipeline in dnceng can't access it. We decide not to authenticate the public pipleline, since this is just a temporary solution until we have official VS builds running. FYI @JoeRobich

Reply:
Since, david has given his thumbs up to the LogHub change, I'll go ahead and merge this one.